<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpDialog</class>
 <widget class="QDialog" name="SignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title_lb">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_lb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_lb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="comfirmPwd_lb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_ledt">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>请输入你的用户名，不超过20位</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_ledt">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入你的密码，至少6位</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="comfirmPwd_ledt">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请再次输入你的密码，两次密码必须一致</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>351</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="signUp_pbt">
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_pbt">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="hintSignup_Info_lb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>title_lb</zorder>
  <zorder>username_lb</zorder>
  <zorder>password_lb</zorder>
  <zorder>comfirmPwd_lb</zorder>
  <zorder>username_ledt</zorder>
  <zorder>password_ledt</zorder>
  <zorder>comfirmPwd_ledt</zorder>
  <zorder>hintSignup_Info_lb</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
